### ISSUE TITLE
[Bug] Constant '[Errno 98] Address in use' errors in docker console

### ISSUE THREAD

**Issue by rursache** [zurdi15/romm#440]
**RomM version**
2.1.0

**Describe the bug**
I just added the romm docker container on my linux machine using this docker run command (i don't use compose):

```
docker run -d \
  --name romm \
  --restart unless-stopped \
  -p 32282:8080 \
  -e TZ=Europe/Bucharest \
  -e IGDB_CLIENT_ID="xxx" \
  -e IGDB_CLIENT_SECRET="yyy" \
  -e ENABLE_RESCAN_ON_FILESYSTEM_CHANGE=true \
  -e ENABLE_SCHEDULED_RESCAN=true \
  -e ENABLE_SCHEDULED_UPDATE_SWITCH_TITLEDB=true \
  -e ENABLE_SCHEDULED_UPDATE_MAME_XML=true \
  -v "/media/radu/Media/Backups/Dumps/ROMs":/romm/library/roms/ \
  -v /home/radu/.romm/resources:/romm/resources \
  -v /home/radu/.romm/config.yml:/romm/config.yml \
  -v /home/radu/.romm/database:/romm/database \
  -v /home/radu/.romm/logs:/romm/logs \
  ghcr.io/zurdi15/romm:latest
```

it starts fine but the moment I tap the scan button (complete rescan checked), it finds some roms then all i see is this in the container logs:

```
2023-11-03T13:57:54.606128994Z WARNING:  [RomM][2023-11-03 15:57:54] Twitch token invalid: fetching a new one...
2023-11-03T13:57:54.761969653Z ERROR:    [Errno 98] Address in use
2023-11-03T13:57:57.124530016Z INFO:     [RomM][alembic.runtime.migration] Context impl SQLiteImpl.
2023-11-03T13:57:57.124556728Z INFO:     [RomM][alembic.runtime.migration] Will assume non-transactional DDL.
```

at that point the romm web service is no longer available (ERR_CONNECTION_REFUSED)

my `config.yaml` if it matters (naming seems like such a pain comparing to emulatorjs tho):
```
system:
  platforms:
    DS: 'nds'
    GBA: 'gba'
    GBC: 'gbc'
    iOS: 'ios'
    N64: 'n64'
    NES: 'nes'
    PS1: 'ps'
    PS2: 'ps2'
    PSP: 'psp'
    SNES: 'snes'
    Switch: 'switch'
    Wii: 'wii'
    Xbox 360: 'xbox360'
```

my roms are in the `/media/radu/Media/Backups/Dumps/ROMs` directory under `Console Name` directory. eg. `/media/radu/Media/Backups/Dumps/ROMs/Xbox 360/some cool game.iso`

**Comment by GAntoine** [comment]
I think we have a fix in the works for this, could you try it with the `dev-2.1.1-rc.1` docker tag?

**Comment by zurdi15** [comment]
Yes, it was fixed in #431

**Comment by rursache** [comment]
thank you, it's indeed fixed in the[dev-latest](https://github.com/users/zurdi15/packages/container/romm/143813975?tag=dev-latest) tag
